appserver/tests/payara-samples/pom.xml

```diff
@@ -108,6 +104,16 @@
                 <artifactId>smallrye-config</artifactId>
                 <version>1.3.7</version>
                 <scope>test</scope>
+                <exclusions>
```

Comment:
This was discovered when we discussed related dependency issues with @MattGill98